src/basic/virt.c

```diff
@@ -148,6 +148,7 @@ static int detect_vm_dmi(void) {
                 int id;
         } dmi_vendor_table[] = {
                 { "KVM",                 VIRTUALIZATION_KVM       },
+                { "Amazon EC2",          VIRTUALIZATION_AMAZON    },
```

Comment:
Please include the link from the commit message here in the comment too (like for VMware below).